website/src/pages/docs/api/error-handler.mdx

```diff
@@ -21,3 +21,26 @@ api.use(
   })
 );
 ```
+
```

Comment:
that's nice
can you also on the top section show an example of the default response response that you'll get if you don't add anything and there is an error?

and also for each example - can you also show the actual full response that would be sent?

Reply:
How about now?

Reply:
nice!
I would also add the full json itself, easier to read

Reply:
I added more details including the expected response fully.